### DISCUSSION TITLE
code:for patches_batch in patch_loader report errors, FileNotFoundError: File not found: "affine".I didn't find out what the problem was。

### DISCUSSION THREAD

**Discussion by boundaryT** [TorchIO-project/torchio#906] [Q&A]
`    for i,file in enumerate(input_files):
        print("data_process" + str(i))
        print(file)
        subject = tio.Subject(source=tio.data.ScalarImage(file))
        grid_sampler = tio.inference.GridSampler(
            subject,  # some NumPy array
            patch_size=(88, 88, 60),
            patch_overlap=4,
        )
        tio.data.inference.aggregator
        patch_loader = torch.utils.data.DataLoader(grid_sampler, batch_size=4)
        print(grid_sampler.subject)
        aggregator = tio.inference.GridAggregator(grid_sampler)
        print(aggregator.patch_overlap)
        with torch.no_grad():
            for patches_batch in patch_loader:
                print(1111111111)
                input_tensor = patches_batch['source'][tio.DATA]
                #test  --not enter model
                print(input_tensor)
                locations = patches_batch[tio.LOCATION]
                logging.info("OK")
                aggregator.add_batch(input_tensor, locations)
        output_tensor = aggregator.get_output_tensor()`

**Comment by fepegar** [comment]
Hi, @boundaryT. Please follow [these instructions](https://github.com/fepegar/torchio/issues/new?assignees=&labels=&template=not_working.yml) to open an issue with detailed information on your problem.
